staging/src/k8s.io/apimachinery/pkg/util/manifest/manifest_test.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubernetes_test
+
+import (
+	"io/ioutil"
+	"testing"
+
+	. "k8s.io/apimachinery/pkg/util/manifest"
+)
+
+func TestLoadAllItemsIntoFlattendList(t *testing.T) {
+	for _, sample := range []struct {
+		path        string
+		expectedLen int
+	}{
+		{
+			path:        "testdata/misc-sample-nested-list-1.json",
+			expectedLen: 6,
```

Comment:
The distance between the data and this expectation makes this test hard to read. I don't have an exact suggestion, maybe add the expectation to the file name or somehow embed the expectation in the file?

Reply:
I've not changes this yet, let's settle the API first :)